tests/platform_tests/cli/test_show_platform.py

```diff
@@ -313,6 +313,11 @@ def test_show_platform_ssdhealth(duthosts, enum_supervisor_dut_hostname):
     @summary: Verify output of `show platform ssdhealth`
     """
     duthost = duthosts[enum_supervisor_dut_hostname]
+    cmd = "cat /sys/block/sda/queue/rotational"
```

Comment:
this is very hardware specific, some platform does not have sda, it is difficult to handle this in the test, can we modify the image to check if disk type is ssd or not?

Reply:
```
lsblk -l -n | grep 'disk' 
```
will list all valid disks in the host